demisto_sdk/commands/validate/tests/GR_validators_test.py

```diff
@@ -0,0 +1,508 @@
+    pack1.content_items.integration.append(
+        mock_integration(name="DeprecatedIntegration", deprecated=True)
+    )
+    pack1.content_items.script.append(
+        mock_script(
+            "SampleScript",
+            [MarketplaceVersions.XSOAR, MarketplaceVersions.MarketplaceV2],
+        )
+    )
+    pack1.content_items.script.append(
+        mock_script(
+            "setIncident",
+            [MarketplaceVersions.XSOAR, MarketplaceVersions.MarketplaceV2],
+        )
+    )
+    pack2.content_items.script.append(mock_script("TestApiModule"))
+    pack2.content_items.script.append(
+        mock_script(
+            "getIncidents",
+            marketplaces=[MarketplaceVersions.XSOAR, MarketplaceVersions.MarketplaceV2],
+            skip_prepare=[SKIP_PREPARE_SCRIPT_NAME],
+        )
+    )
+    pack2.content_items.classifier.append(mock_classifier("SampleClassifier2"))
+    pack2.content_items.test_playbook.append(mock_test_playbook())
+    pack3.content_items.playbook.append(
+        mock_playbook(
+            "SamplePlaybook",
+            [MarketplaceVersions.XSOAR, MarketplaceVersions.XPANSE],
+            "6.5.0",
+            GENERAL_DEFAULT_FROMVERSION,
+        )
+    )
+    pack3.content_items.playbook.append(
+        mock_playbook(
+            "SamplePlaybook2",
+            [MarketplaceVersions.XSOAR],
+            GENERAL_DEFAULT_FROMVERSION,
+            "6.5.0",
+        )
+    )
+    pack3.content_items.script.append(mock_script("SampleScript2"))
+    pack3.content_items.script.append(
+        mock_script(
+            "setAlert", [MarketplaceVersions.XSOAR, MarketplaceVersions.MarketplaceV2]
+        )
+    )
+    pack3.content_items.script.append(
+        mock_script(
+            "getAlert", [MarketplaceVersions.XSOAR, MarketplaceVersions.MarketplaceV2]
+        )
+    )
+    pack3.content_items.script.append(
+        mock_script(
+            "getAlerts", [MarketplaceVersions.XSOAR, MarketplaceVersions.MarketplaceV2]
+        )
+    )
+    pack4.content_items.playbook.append(mock_playbook("SamplePlaybook"))
+    repository.packs.extend([pack1, pack2, pack3, pack4])
+    mocker.patch(
+        "demisto_sdk.commands.content_graph.content_graph_builder.ContentGraphBuilder._create_content_dto",
+        return_value=repository,
+    )
+    return repository
+
+
+def test_MarketplacesFieldValidator_is_valid(repository: ContentDTO):
+    """
+    Given
+    - A content repo
+    When
+    - running MarketplacesFieldValidator is_valid function.
+    Then
+    - Validate the existence of invalid marketplaces usages.
+    """
+
+    graph_interface = ContentGraphInterface()
+    create_content_graph(graph_interface)
+    MarketplacesFieldValidator.graph_interface = graph_interface
+    validation_results = MarketplacesFieldValidator().is_valid(repository.packs)
+    validation_results_messages = {
+        validation_result.message for validation_result in validation_results
+    }
```

Comment:
Let's validate using 'in'.
Also, let's validate the expected length of the result.

Reply:
done